impose/gui/collect.py

```diff
@@ -79,6 +80,12 @@ def add_paths(self, paths):
         for pp in paths:
             try:
                 self.session_scheme.append(pp)
+            except BmlabInvalidFileError as e:
```

Comment:
There should be no data-file-format-specific stuff in collect.py. Can you handle this without dealing with this here?